spec/features/user_management_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+require_relative 'helpers/session'
+
+include SessionHelpers
+
+feature 'User signs up' do
```

Comment:
Again, your scenarios here should describe to the reader the behaviour we would expect given certain conditions...
So something like:
- Feature: A user is welcomed to Chitter
  - Scenario: when signed up with correct details
- Feature: A user is prompted to re-enter details
  - Scenario: having submitted non-matching passwords.
  - Scenario: having submitted an email address already in use.  

Not totally convinced that is best practice but I think it's an improvement.